web-common/src/features/dashboards/actions/index.ts

```diff
@@ -0,0 +1,41 @@
+import { get } from "svelte/store";
+import type { BusinessModel } from "../business-model/business-model";
+import { cancelDashboardQueries } from "../dashboard-queries";
+import { metricsExplorerStore } from "../dashboard-stores";
+
+export function clearFilterForDimension(
+  ctx: BusinessModel,
+  dimensionId,
+  include: boolean
+) {
+  const metricViewName = get(ctx.metricsViewName);
+  cancelDashboardQueries(ctx.queryClient, metricViewName);
+  metricsExplorerStore.clearFilterForDimension(
```

Comment:
Should the `metricsExplorerStore` come from the business model context?

Reply:
Yea actually all it really contains at this point is actions, and I think we should just split those out into separate action functions in the actions file.

I didn't do that yet here to limit the scope of changes, but can try adding that as well

Reply:
If we do split it, it would perhaps make sense to move the function body here wrapped with an update.

Reply:
Done